docs/index.md

```diff
@@ -1,24 +1,11 @@
 <img src="img/logo.png" width="200">
 
-# CFripper
+# CFRipper
 
 [![Build Status](https://travis-ci.org/Skyscanner/cfripper.svg?branch=master)](https://travis-ci.org/Skyscanner/cfripper)
 [![PyPI version](https://badge.fury.io/py/cfripper.svg)](https://badge.fury.io/py/cfripper)
 
-Lambda function to "rip apart" a CloudFormation template and check it for security compliance.
-
-## Sample pipeline with CFripper
-
-CFripper is a Python tool that aims to prevent vulnerabilities from getting to production infrastructure through vulnerable CloudFormation scripts. As with the other security tools that we use at Skyscanner, CFripper is part of the CI/CD pipeline. It runs just before a CloudFormation stack is deployed or updated and if the CloudFormation script fails to pass the security check it fails the deployment and notifies the team that owns the stack. This is an example of how you might set up CFripper as an AWS Lambda:
```

Comment:
Are we using `CFRipper` as in line 3 or `CFripper` as in line 12?

Reply:
The official name is `CFRipper`